docs/Layout.md

```diff
@@ -236,4 +236,45 @@ This property accepts the following subclasses:
 
 To override the style of `<Layout>` using the [MUI style overrides](https://mui.com/customization/theme-components/), use the `RaLayout` key.
 
-**Tip**: If you find yourself tweaking the `<Layout sx>` prop, you're probably better off [writing your own layout component](./Theming.md#layout-from-scratch). 
\ No newline at end of file
+**Tip**: If you need to override global styles (like the default font size or family), you should [write a custom theme](./Theming.md#theming) rather than override the `<Layiyt sx>` prop. And you you need to tweak the default layout to add a right column or move the menu to the top, you're probably better off [writing your own layout component](./Theming.md#layout-from-scratch). 
```

Comment:
```suggestion
**Tip**: If you need to override global styles (like the default font size or family), you should [write a custom theme](./Theming.md#theming) rather than override the `<Layout sx>` prop. And if you need to tweak the default layout to add a right column or move the menu to the top, you're probably better off [writing your own layout component](./Theming.md#layout-from-scratch). 
```